docs/data-ingestion/Proxmox.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Proxmox
+weight: 1
+sort: 1
+menu:
+  docs:
+    identifier: "proxmox"
+    parent: "data-ingestion"
+    weight: 1
+    # sort: 1
+aliases:
+  - /data-ingestion/proxmox.html
+  - /data-ingestion/Proxmox.html
+---
+
+# Proxmox Data Ingestion
```

Comment:
```suggestion
```
this anchor will make the link look like `/data-ingestion/proxmox#proxmox-data-ingestion,` which is almost a palindrome. Shall we just rm this anchor?